sql/core/src/test/scala/org/apache/spark/sql/streaming/StreamSuite.scala

```diff
@@ -976,24 +976,48 @@ class StreamSuite extends StreamTest {
       CheckAnswer(1 to 3: _*))
   }
 
-  test("streaming limit in complete mode") {
+  test("streaming limit before agg in complete mode (SPARK-30658)") {
```

Comment:
@tdas this is not the right style. It should be `test("SPARK-12345: a short description of the test")`. Could you also fix other test names?